ui/components/component-library/modal-header/deprecated/modal-header.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
Moving current `ModalHeader` to deprecated folder will replace these deprecated components in a subsequent PR